app/views/content/train-to-be-a-teacher/promos/_find-your-course.html.erb

```diff
@@ -8,8 +8,7 @@
   <div class="right">
     <span class="caption-l caption--bold caption--purple">Find your course</span>
     <p>
-      You can search for postgraduate courses that lead to QTS (not including
-      Teach First).
+      You can search for postgraduate courses that lead to qualified teacher status (QTS), or find Teach First courses on their website.
```

Comment:
Not sure this entirely makes sense now re Teach First courses, particularly as it doesn't have a link. We really want people going through to Find? Do we need to mention Teach First here - is it confusing? We have a link to their site further up the page.

Happy for you to disagree though @EllieNodder !

Reply:
I'd suggest making it clearer in the Teach First section and removing reference here - but happy for you to disagree from a content perspective @EllieNodder !